<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" ignored="0" total="6" passed="4">
  <reporter-output>
  </reporter-output>
  <suite name="NopCommerceV1 Automation Suite" duration-ms="208705" started-at="2021-10-14T19:26:20 IST" finished-at="2021-10-14T19:29:49 IST">
    <groups>
    </groups>
    <test name="Nop CommerceV1 Regression " duration-ms="208705" started-at="2021-10-14T19:26:20 IST" finished-at="2021-10-14T19:29:49 IST">
      <class name="com.nopcommerce.tests.LoginTests">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.nopcommerce.tests.LoginTests@4b553d26]" name="setUp" is-config="true" duration-ms="52231" started-at="2021-10-14T19:26:21 IST" finished-at="2021-10-14T19:27:13 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="loginTest()[pri:0, instance:com.nopcommerce.tests.LoginTests@4b553d26]" name="loginTest" duration-ms="28062" started-at="2021-10-14T19:27:13 IST" finished-at="2021-10-14T19:27:41 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.nopcommerce.tests.LoginTests@4b553d26]" name="tearDown" is-config="true" duration-ms="1612" started-at="2021-10-14T19:27:41 IST" finished-at="2021-10-14T19:27:43 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.nopcommerce.tests.LoginTests@4b553d26]" name="setUp" is-config="true" duration-ms="11854" started-at="2021-10-14T19:27:47 IST" finished-at="2021-10-14T19:27:59 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="loginTest_DataDriven(java.lang.String,java.lang.String)[pri:0, instance:com.nopcommerce.tests.LoginTests@4b553d26]" name="loginTest_DataDriven" duration-ms="11940" started-at="2021-10-14T19:27:59 IST" data-provider="getData" finished-at="2021-10-14T19:28:11 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[admin@yourstore.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[admin]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest_DataDriven -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.nopcommerce.tests.LoginTests@4b553d26]" name="tearDown" is-config="true" duration-ms="1240" started-at="2021-10-14T19:28:11 IST" finished-at="2021-10-14T19:28:12 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.nopcommerce.tests.LoginTests@4b553d26]" name="setUp" is-config="true" duration-ms="11731" started-at="2021-10-14T19:28:12 IST" finished-at="2021-10-14T19:28:24 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="loginTest_DataDriven(java.lang.String,java.lang.String)[pri:0, instance:com.nopcommerce.tests.LoginTests@4b553d26]" name="loginTest_DataDriven" duration-ms="4109" started-at="2021-10-14T19:28:24 IST" data-provider="getData" finished-at="2021-10-14T19:28:28 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[admin@yourstore.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[test]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at com.nopcommerce.tests.LoginTests.loginTest_DataDriven(LoginTests.java:69)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest_DataDriven -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.nopcommerce.tests.LoginTests@4b553d26]" name="tearDown" is-config="true" duration-ms="835" started-at="2021-10-14T19:28:28 IST" finished-at="2021-10-14T19:28:29 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.nopcommerce.tests.LoginTests@4b553d26]" name="setUp" is-config="true" duration-ms="12472" started-at="2021-10-14T19:28:29 IST" finished-at="2021-10-14T19:28:41 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="loginTest_DataDriven(java.lang.String,java.lang.String)[pri:0, instance:com.nopcommerce.tests.LoginTests@4b553d26]" name="loginTest_DataDriven" duration-ms="2578" started-at="2021-10-14T19:28:41 IST" data-provider="getData" finished-at="2021-10-14T19:28:44 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[admin1@yourstore.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[admin]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at com.nopcommerce.tests.LoginTests.loginTest_DataDriven(LoginTests.java:69)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest_DataDriven -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.nopcommerce.tests.LoginTests@4b553d26]" name="tearDown" is-config="true" duration-ms="753" started-at="2021-10-14T19:28:44 IST" finished-at="2021-10-14T19:28:44 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.nopcommerce.tests.LoginTests@4b553d26]" name="setUp" is-config="true" duration-ms="12318" started-at="2021-10-14T19:28:44 IST" finished-at="2021-10-14T19:28:57 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="titleTest()[pri:0, instance:com.nopcommerce.tests.LoginTests@4b553d26]" name="titleTest" duration-ms="15" started-at="2021-10-14T19:28:57 IST" finished-at="2021-10-14T19:28:57 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- titleTest -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.nopcommerce.tests.LoginTests@4b553d26]" name="tearDown" is-config="true" duration-ms="1189" started-at="2021-10-14T19:28:57 IST" finished-at="2021-10-14T19:28:58 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.nopcommerce.tests.LoginTests -->
      <class name="com.nopcommerce.tests.AddNewCustomerTest">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.nopcommerce.tests.AddNewCustomerTest@45820e51]" name="setUp" is-config="true" duration-ms="22678" started-at="2021-10-14T19:28:58 IST" finished-at="2021-10-14T19:29:21 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="newCustomerTest()[pri:0, instance:com.nopcommerce.tests.AddNewCustomerTest@45820e51]" name="newCustomerTest" duration-ms="27295" started-at="2021-10-14T19:29:21 IST" finished-at="2021-10-14T19:29:48 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- newCustomerTest -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.nopcommerce.tests.AddNewCustomerTest@45820e51]" name="tearDown" is-config="true" duration-ms="1214" started-at="2021-10-14T19:29:48 IST" finished-at="2021-10-14T19:29:49 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.nopcommerce.tests.AddNewCustomerTest -->
    </test> <!-- Nop CommerceV1 Regression  -->
  </suite> <!-- NopCommerceV1 Automation Suite -->
</testng-results>
